components/developer/rust/Makefile

```diff
@@ -36,6 +36,7 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		rustc
 COMPONENT_VERSION=	1.89.0
+COMPONENT_REVISION=	1
 COMPONENT_SUMMARY=	Rust - Safe, concurrent, practical language
```

Comment:
There is no need to set COMPONENT_REVISION as the original PR has not yet successully been published.